feat(docs): add spaces object storage in digitalocean-droplet doc (#1279)
kestra-io/docs: content/docs/02.installation/11.digitalocean-droplet.md

depends_on:
  postgres:
    condition: service_started
```

In order for the changes to take effect, restart the docker services with `sudo docker compose restart` or `sudo docker compose up -d`.

## Configure Spaces Object Storage

By default, internal storage is implemented using the local file system. This section will guide you on how to change the storage backend to Spaces Object Storage to ensure more reliable, durable, and scalable storage.

Firstly, we will create the access key and secret key that can be used to connect to Spaces Object Storage.

1. Navigate to the `API` from the left navigation menu.
2. Go to the `Spaces Keys` tab.
3. Click on `Generate New Key` button.
4. Provide an appropriate name for the spaces access key, and click on `Create Access Key`.

> 5. A new access key with the given name will be generated. Take a note of the secret key as you would not be able to retriev it later.

Reviewer: ```suggestion
5. A new access key with the given name will be generated. Take a note of the secret key as you will not be able to retrieve it later.
```

![spaces_api1](/docs/administrator-guide/deployment/digitalocean-droplet/spaces_api1.png)

![spaces_api2](/docs/administrator-guide/deployment/digitalocean-droplet/spaces_api2.png)

![spaces_api3](/docs/administrator-guide/deployment/digitalocean-droplet/spaces_api3.png)

> Let us now proceed to create a bucket in the Spaces Object Storage.

Reviewer: ```suggestion
Let's create a bucket in the Spaces Object Storage.
```
